skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -1015,5 +1042,82 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestAppend(unittest.TestCase):
```

Comment:
I like this style of testing: a test class for each method. It doesn't match the style used in the rest of the tests though -- @ebolyen what do you think about mixing styles of testing here? I'm on the fence.